app/assets/stylesheets/publisher.scss

```diff
@@ -29,25 +29,37 @@
     }
 
     .options_and_submit {
-      padding: 10px 0 ;
-      #publisher_service_icons {
-        .btn-link{ text-decoration: none; }
-        .btn-link.question_mark .entypo-cog {
+      padding: 10px 0;
+      .btn{ height: 32px; }
+      #publisher-service-icons {
+        text-decoration: none;
+        .entypo-cog, .service_icon {
           color: $text-grey;
           font-size: 16px;
-          line-height: $line-height-computed;
-        }
-        .btn-link.question_mark:hover.entypo-cog { color: $black; }
-        .btn-link.service_icon {
-          padding-left: 5px;
-          padding-right: 5px;
+          line-height: 16px;
```

Comment:
Before this PR the line-height was 18px for this button vs. 18.57px for other buttons. `$line-height-computed` is the line-height Bootstrap uses when a unit-less line-height could not be used. With your PR this is 16px vs. 18.57px.